platforms/documentation/docs/src/docs/userguide/dep-man/declaring-repositories/supported_repository_types.adoc

```diff
@@ -0,0 +1,160 @@
+For instance, if Gradle finds `jmxri-1.2.1.jar` in a flat directory and `jmxri-1.2.1.pom` in another repository, it will use the metadata from the latter.
+
+If you need to override remote artifacts with local ones, consider using an Ivy or Maven repository pointing to a local directory instead.
+
+[[sec:maven_repo]]
+== Maven repositories
+
+Many organizations host dependencies in private Maven repositories accessible within their network.
+Gradle can declare Maven repositories by specifying their URL:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=maven-like-repo]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=maven-like-repo]"]
+====
+
+[[sub:custom-maven-repo]]
+=== Composite Maven repository
+
+Sometimes, POMs are published in one location, and JARs in another.
+You can define such a repository as follows:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=maven-like-repo-with-jar-repo]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=maven-like-repo-with-jar-repo]"]
+====
+
+Gradle will first look for POMs and artifacts at the base URL, and if the artifact is not found, it will check the additional `artifactUrls`.
+
+[[sub:accessing_secured_maven_repositories]]
+=== Authenticated Maven repository
+
+You can specify credentials for Maven repositories that require authentication.
+See <<supported_repository_protocols.adoc#supported_transport_protocols>> for authentication options.
+
+[[sub:maven_local]]
+=== Local Maven repository
+
+Gradle can also consume dependencies from the link:https://maven.apache.org/guides/introduction/introduction-to-repositories.html[local Maven repository].
+This is useful for teams that publish locally with one project and consume the artifacts with another.
+To declare the local Maven cache as a repository:
+
+====
+include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=maven-local]"]
+include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=maven-local]"]
+====
+
+NOTE: Gradle stores resolved dependencies in <<dependency_resolution.adoc#sec:dependency_cache,its own cache>>.
+A build does not need to declare the local Maven repository even if you resolve dependencies from a Maven-based, remote repository.
+
+WARNING: Before adding Maven local as a repository, you should <<declaring_repositories_basics.adoc#sec:case-for-maven-local, make sure this is really required>>.
+
+Gradle uses the same logic as Maven to determine the location of your local Maven cache.
+It prioritizes the `settings.xml` in the user’s home directory,`<user_home>/.m2`, over the one in `__M2_HOME__/conf`.
+If no `settings.xml` is found, Gradle uses the default location at `<user_home>/.m2/repository`.
```

Comment:
We've already described a lot of this info on the other page. I'd recommend we avoid duplicating it and either link from that page to this or the other way around

Reply:
Removed.